searx/engines/packagist.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+# pylint: disable=missing-function-docstring
+"""Packagist (IT)
+
+"""
+
+from json import loads
+from urllib.parse import urlencode
+from dateutil import parser
+
+import requests
+
+from searx import logger
+
+about = {
+     "website": 'https://packagist.org',
+     "wikidata_id": 'Q108311377',
+     "official_api_documentation": 'https://packagist.org/apidoc',
+     "use_official_api": True,
+     "require_api_key": False,
+     "results": 'JSON',
+ }
+
+categories = ['it']  # optional
+paging = True
+
+base_url = "https://packagist.org/"
+search_url = base_url + "search.json?{query}&per_page=25"
+
+def metadata(package_url):
+    result = {}
+    body = loads(requests.get(package_url + ".json").text)
+
+    package = body.get("package")
```

Comment:
@koelle25 what do you think, does it make sense to collect some more additional data (and ad it to the `result["content"]`) .. e.g. `repository`.

![grafik](https://user-images.githubusercontent.com/554536/131479341-b04104af-24de-4dca-901e-fb6ceafc1ae9.png)